tests/performance/Taskfile.yaml

```
@@ -0,0 +1,21 @@
+# https://taskfile.dev
+
+version: "3"
+
+silent: true
+
+tasks:
+  help:
+    desc: "Help about bootstrapper.sh."
+    cmds:
+      - ./bootstrapper.sh --help
+
+  generate:
+    desc: "Generate virtual machines."
+    cmds:
+      - ./bootstrapper.sh generate --count=1
```

```suggestion
      - ./bootstrapper.sh generate --count={{ .TaskArgs.count | default "1" }}
```

task generate --count=10